sycl/test/graph/graph-explicit-dotp.cpp

```diff
@@ -1,94 +1,97 @@
-#include <iostream>
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
 #include <CL/sycl.hpp>
+#include <iostream>
 
 #include <sycl/ext/oneapi/experimental/graph.hpp>
 
 const size_t n = 10;
 
 float host_gold_result() {
-    float alpha = 1.0f;
-    float beta = 2.0f;
-    float gamma = 3.0f;
-    
-    float sum = 0.0f;
-    
-    for(size_t i = 0; i < n; ++i) {
-        sum += (alpha * 1.0f + beta * 2.0f)
-             * (gamma * 3.0f + beta * 2.0f);
-    }
-    
-    return sum;
+  float alpha = 1.0f;
+  float beta = 2.0f;
+  float gamma = 3.0f;
+
+  float sum = 0.0f;
+
+  for (size_t i = 0; i < n; ++i) {
+    sum += (alpha * 1.0f + beta * 2.0f) * (gamma * 3.0f + beta * 2.0f);
+  }
+
+  return sum;
 }
 
 int main() {
-    float alpha = 1.0f;
-    float beta = 2.0f;
-    float gamma = 3.0f;
-    
-    float *x , *y, *z;
+  float alpha = 1.0f;
+  float beta = 2.0f;
+  float gamma = 3.0f;
 
-    sycl::property_list properties{
+  sycl::property_list properties{
       sycl::property::queue::in_order(),
```

Comment:
We might want to use curly braces here too for consistency.
```suggestion
      sycl::property::queue::in_order{},
```